app/src/test/java/edu/wpi/first/shuffleboard/app/json/WidgetSaverTest.java

```diff
@@ -22,7 +21,7 @@
 
   @Override
   public void start(Stage stage) throws Exception {
-    Widgets.discover();
+    // Just here so can run on the FX thread
```

Comment:
Seems weird that we have code that needs to run on the FX Thread without using a stage.  Not something to fix in this PR but we should look into that.